projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/DataJobDeploymentCrudIT.java

```diff
@@ -225,6 +225,47 @@ public void testDataJobDeploymentCrud() throws Exception {
       cronJob = cronJobOptional.get();
       Assertions.assertEquals(false, cronJob.getEnabled());
 
+      // Execute set vdk version for deployment
+      mockMvc.perform(patch(String.format("/data-jobs/for-team/%s/jobs/%s/deployments/%s",
+                      TEST_TEAM_NAME,
+                      TEST_JOB_NAME,
+                      DEPLOYMENT_ID))
+                      .with(user("user"))
+                      .content(getDataJobDeploymentVdkVersionRequestBody("new_vdk_version_tag"))
+                      .contentType(MediaType.APPLICATION_JSON))
+              .andExpect(status().isAccepted());
+
+      // verify vdk version is returned
+      mockMvc.perform(get(String.format("/data-jobs/for-team/%s/jobs/%s/deployments/%s",
+                      TEST_TEAM_NAME,
+                      TEST_JOB_NAME,
+                      DEPLOYMENT_ID))
+                      .with(user("user"))
+                      .contentType(MediaType.APPLICATION_JSON))
+              .andExpect(status().isOk())
+              .andExpect(jsonPath("$.vdk_version", is("new_vdk_version_tag")));
+
+      // Execute reset back vdk version for deployment
+      mockMvc.perform(patch(String.format("/data-jobs/for-team/%s/jobs/%s/deployments/%s",
+                      TEST_TEAM_NAME,
+                      TEST_JOB_NAME,
+                      DEPLOYMENT_ID))
+                      .with(user("user"))
+                      .content(getDataJobDeploymentVdkVersionRequestBody(""))
+                      .contentType(MediaType.APPLICATION_JSON))
+              .andExpect(status().isAccepted());
+
+      // verify vdk version is reset correctly
+      mockMvc.perform(get(String.format("/data-jobs/for-team/%s/jobs/%s/deployments/%s",
+                      TEST_TEAM_NAME,
+                      TEST_JOB_NAME,
+                      DEPLOYMENT_ID))
+                      .with(user("user"))
+                      .contentType(MediaType.APPLICATION_JSON))
+              .andExpect(status().isOk())
+              .andExpect(jsonPath("$.vdk_version", is("release")));
```

Comment:
Could you describe why defaulting to release? What is the expected result when setting the version to empty value

Reply:
Explained it in a comment : 

```

      // by default the version is the same as the tag specified by datajobs.vdk.image
      // for integration test this is registry.hub.docker.com/versatiledatakit/quickstart-vdk:release
```